server/sonar-server/src/main/java/org/sonar/server/computation/container/ComputeEngineContainerImpl.java

```diff
@@ -154,14 +167,30 @@ private static List componentClasses() {
       NewCoverageMetricKeysModule.class,
 
       // issues
-      ScmAccountCacheLoader.class,
-      ScmAccountCache.class,
-      SourceLinesCache.class,
-      IssueComputation.class,
-      RuleCache.class,
       RuleCacheLoader.class,
+      RuleRepositoryImpl.class,
+      ScmAccountToUserLoader.class,
```

Comment:
what about using some modules to group components used in a single other component with that component? or to group all debt related components?
would lighten this class and be more resistant to unappropriate change (could even be unit tested)